book/5-data-mgmt-exp-proc/eda-cv-tasks.md

```diff
@@ -0,0 +1,117 @@
+
+The data split refers to how one might partition the data for training/validating/testing the model. Generally, all sets will need equal proportions of the bbox size (small/medium/large) and class distribution. In the ideal scenario, these proportions are easy to split at random, but it is extremely rare. Thus, by performing EDA on the annotations, the Engineer will have a better idea on how the data split can be done. 
+
+With these considerations in mind, these are some guiding questions when performing EDA on annotations: 
+1. How many instances of each classes are there?
+1. Bounding Box - what is width and height distribution overall and per class?
+1. Bounding Box - what is the overall size (W*H) overall and per class?
+1. Bounding Box - What are the top few Width/Height combination? This can be found using clustering methods.
+1. Segmentation - what is distribution of the mask pixel counts overall and per class?
+
+### Colours of Annotations
+As with images, the annotations have 1 or more channels per pixel, with the most popular format being Red, Green and Blue (RGB). The utility for this mainly lets the researchers know if the model would be better served trying to use certain channels when making inference. 
+
+#### Average Intensity per channel  
+To explore this for all classes, we can firstly average the colour intensity of the annotation's pixels per channel.  
+
+Secondly, countplot of the average colour intensity values could be generated. The image below shows the counts of each annotation with the average values of the red channels separated by classes. In this case, there are three classes, with one of the class having very spread out red channel intensity, and the other two classes having colour intensity being normally distributed.
+
+![title](../assets/images/screenshots/red_channel_average_counts.png)
+
+#### Mode Intensity per channel    
+
+Besides averaging the intensity per annotation, the highest occurring value (mode) of the intensity of the annotation can also be used to analyse the colour of the annotation.  
+
+![title](../assets/images/screenshots/red_channel_dominant_counts.png)
+
+There are many other ways to visualise this. The main goal is to check on the colour distribution among the annotations. 
+
+## Annotations Random Sample Checks
+
+### Bounding Box and Mask Random Sample Checks
+
+Bounding Boxes Random Sample Check refers to performing a check by sampling images and analysing their annotations. These try to account to account for human error that very often occurs. 
+
+1. Are the bboxes/masks drawn tight enough? 
+1. Were the bboxes/masks drawn accurately?
+1. Are the objects being occluded? In other words, were parts of the object blocked by another object?
```

Comment:
How about adding this pointer to point 3?
`In general, if the object of interest is being occluded, its annotation should still be drawn such that it includes the occluded portion.`